### PR TITLE
feat(id_token): Include id_token to omniauth payload

### PR DESCRIPTION
**Pull request by yannvery**
Include the id_token attributes to the Omniauth payload.

### CODE REVIEW

File: test/lib/omniauth/strategies/openid_connect_test.rb [hunk 1]
```diff
@@ -343,7 +360,7 @@ def test_info
       end
 
       def test_extra
-        assert_equal({ raw_info: user_info.as_json }, strategy.extra)
+        assert_equal({ raw_info: user_info.as_json, id_token: id_token }, strategy.extra)
```
Comment (giniouxe) [inline test/lib/omniauth/strategies/openid_connect_test.rb:363]: No `.as_json` for the `id_token`?

**Review by giniouxe — COMMENTED** [review]
One question.

File: test/lib/omniauth/strategies/openid_connect_test.rb (hunk 1 above)
Comment (yannvery) [reply test/lib/omniauth/strategies/openid_connect_test.rb:363]: @giniouxe `user_info.as_json` has an odd name, in fact this returns a hash (https://www.rubydoc.info/gems/openid_connect/OpenIDConnect/ConnectObject#as_json-instance_method)

`id_token` already returns a hash.

**Review by giniouxe — APPROVED** [review]
:+1:

**Review by eraleis — APPROVED** [review]

**Pull request merged**